packages/framework-core/src/booster-event-dispatcher.ts

```diff
@@ -36,8 +43,8 @@ export class BoosterEventDispatcher {
       switch (eventEnvelope.kind) {
         case 'event':
           logger.debug('[BoosterEventDispatcher#eventProcessor]: Started processing workflow for event:', eventEnvelope)
-          // TODO: Separate into two independent processes the snapshotting/read-model generation process from the event handling process
-          await Promise.allSettled([
+          // TODO: Separate into two independent processes the snapshotting/read-model generation process from the event handling process`
+          await Promises.allSettledAndFulfilled([
```

Comment:
Here we were waiting for a `throw` when something was wrong, but `allSettled` never throws. Check the documentation of `allSettledAndFulfilled` to know more (in this PR)